src/hooks/journals/useJournalData.ts

```diff
@@ -35,18 +32,28 @@ const useJournalsForCollection = (collectionName: string | undefined) => {
     }, [gatewayConfig]);
 
     const fetcher = useCallback(
-        (_url: string) => {
+        async (_url: string) => {
             if (journalClient && collectionName) {
                 const journalSelector = new JournalSelector().collection(
                     collectionName
                 );
-                return journalClient.list(journalSelector).then((result) => {
-                    const journals = result.unwrap();
 
-                    return {
-                        journals: journals.length > 0 ? journals : [],
-                    };
-                });
+                const dataPlaneListResponse = await dataPlaneFetcher_list(
+                    journalClient,
+                    journalSelector,
+                    'JournalData'
+                );
+
+                if (!Array.isArray(dataPlaneListResponse)) {
+                    return Promise.reject(dataPlaneListResponse);
+                }
+
+                return {
+                    journals:
+                        dataPlaneListResponse.length > 0
+                            ? dataPlaneListResponse
+                            : [],
+                };
```

Comment:
Made these use identical variable names to make merging these together later easier.

I was going to just have the array and response in the list call but ran into typescript issues and did not want to keep trying to wrangle those.